Helper/Config.php

```diff
@@ -808,10 +808,30 @@ public function isSandboxModeSet($store = null)
         );
     }
 
+    /**
+     * @param $path
+     * @param $default
+     * @return mixed
+     */
     public function getCustomURLValueOrDefault($path, $default)
     {
         $storedValue = $this->getScopeConfig()->getValue($path);
-        return !empty($storedValue) ? $storedValue : $default;
+
+        return $this->validateCustomUrl($storedValue) ? $storedValue : $default;
+    }
+
+    /**
+     * @param $url
+     * @return bool
+     */
+    protected function validateCustomUrl($url) {
+        if (!$url) {
+            return false;
+        }
+
+        $last7Characters = substr($url, -7);
+        $last8Characters = substr($url, -8);
+        return ($last7Characters == 'bolt.me' || $last8Characters == 'bolt.me/');
```

Comment:
I don't believe custom URLs are meant to work from bolt subdomains only. More like from ngrok, as I get it. @roopakv ?

Reply:
@danac-gs seems like this is something that was requested by a emrchant. even though these are used only in sandbox mode and when set they wanted this.

but also note we have a pro ngrok accoutn which points to *.dev.bolt.me. We should make sure you have access to one.